client/src/main/java/io/pravega/client/control/impl/ControllerImpl.java

```diff
@@ -179,6 +179,7 @@ public ControllerImpl(final ControllerImplConfig config,
         this(NettyChannelBuilder.forTarget(config.getClientConfig().getControllerURI().toString())
                                 .nameResolverFactory(new ControllerResolverFactory(executor))
                                 .defaultLoadBalancingPolicy("round_robin")
+                                .directExecutor()
```

Comment:
Observing that `grpc-default-worker-ELG-*` threads created even with this changeset. The number of threads is 2 * available processors .

Reply:
Sorry, I wrote the wrong thing in the issue. This removes the `grpc-default-executor` threads from the client.